maintenance/aws-janitor/main.go

```diff
@@ -466,6 +486,10 @@ func (internetGateways) MarkAndSweep(sess *session.Session, acct string, region
 					glog.Warningf("%v: detach from %v failed: %v", i.ARN(), *att.VpcId, err)
 				}
 			}
+			if isDefault {
+				glog.Infof("%s: skipping delete %T: %v", i.ARN(), ig, ig)
```

Comment:
I think we will see the same duplicated message if isDefault is true, because of line 478.
Is that intentional?

Reply:
More context please

Reply:
I dropped one of the messages